charts/generic-govuk-app/templates/service.yaml

```diff
@@ -20,4 +20,5 @@ spec:
       port: {{ .Values.service.port }}
       targetPort: {{ .Values.nginxPort }}
   selector:
-    app: {{ $fullName }}
+    # app: {{ $fullName }}
```

Comment:
nit: remove commented-out line (or if you intended to keep it as a reminder to remove the condition later or something, then change it into a TODO)
```suggestion
```